src/qt/coincontroldialog.cpp

```diff
@@ -117,6 +117,8 @@ CoinControlDialog::CoinControlDialog(QWidget *parent) :
     // (un)select all
     connect(ui->pushButtonSelectAll, SIGNAL(clicked()), this, SLOT(buttonSelectAllClicked()));
 
+    ui->treeWidget->headerItem()->setText(COLUMN_CHECKBOX, QString());
```

Comment:
little nit: wouldn't it be good to add a comment above this line saying that this is a workaround added because of a Qt4 bug? Maybe in far future we once drop Qt4 support and should remove such "hacks"